tests/preprocessing/test_target_cropper.py

```diff
@@ -34,9 +34,15 @@ def test_target_cropper(image_name: str, target_id: str, expected_name: str) ->
     expected_name : str
         Name of the expected file used to verify the crop.
     """
-    image_path = Path(PAINT_ROOT) / "tests" / "data" / "test_data" / f"{image_name}.png"
+    image_path = (
+        Path(PAINT_ROOT) / "tests" / "preprocessing" / "test_data" / f"{image_name}.png"
```

Comment:
Not sure about this path. I think it refers to the markers so it should be changed like:
```suggestion
        Path(PAINT_ROOT) / "tests" / "data" / "test_data" / f"{image_name}.png"
```

Reply:
Nope, this is correct since the test package also now has the name "preprocessing"